pkg/handlers/scripts/install.sh

```diff
@@ -105,7 +142,24 @@ function install {
 	fi
 	#move into PATH or cwd
 	chmod +x $TMP_BIN || fail "chmod +x failed"
-	mv $TMP_BIN $OUT_DIR/kubectl-$PROG 2>/dev/null || sudo mv $TMP_BIN $OUT_DIR/kubectl-$PROG || fail "mv failed" #FINAL STEP!
+	# read installation path
+	read_installation_path
+	echo "Installing to $OUT_DIR"
+	[ ! -d $OUT_DIR ] && fail "output directory missing: $OUT_DIR"
+	if [ ! -w $OUT_DIR ]; then 
+		echo "You don't have write permissions to $OUT_DIR"
+		echo "Would you like to enter your password to grant write permissions"
+		echo "to $OUT_DIR? [y/n]"
```

Comment:
Made a wording change. Does it need to be on two lines? Also, does the user's response end up on the next line? From the examples it looks that way, which is odd to me.
```suggestion
		echo "Do you want to enter your password to grant write permissions to $OUT_DIR [Y/n]?"
```